A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -754,9 +762,13 @@ class CDUPerformancePage {
                 CDUPerformancePage.ShowAPPRPage(mcdu);
             }
         };
-        let transAltCell = "---";
+
+        let transAltCell = "[\xa0][color]cyan";
         if (isFinite(mcdu.perfApprTransAlt)) {
             transAltCell = mcdu.perfApprTransAlt.toFixed(0);
+            if (!mcdu.perfApprTransAltPilotEntered) {
+                transAltCell += "[s-text]";
+            }
         }
         mcdu.onLeftInput[3] = (value) => {
             if (mcdu.setPerfApprTransAlt(value)) {
```

Comment:
This needs to follow the logic from the TAKEOFF page: 0-45000ft, rounded to nearest 10ft.

Reply:
Takeoff page is 1000 - 45000, nearest 10. I've taken that logic (will push maybe this evening when I get a chance to work on the other bits).